### PR TITLE
Pr takeoff landing (squashed and rebased)

### PR DESCRIPTION
**Pull request by julianoes**
This replaces #6035.

@MaEtUgR @Stifael @tumbili please sanity check, it should be almost identical to #6035.

@MaEtUgR please quickly test in SITL.

### CODE REVIEW

**Comment by LorenzMeier** [comment]
Please squash the commits on merge. This should be one commit under Roman's name with Julian as the rebaser . Thanks!

**Comment by tumbili** [comment]
@LorenzMeier @julianoes Going for this with the bebop. Kind of weird to takeoff and land manually with a joystick.

**Comment by julianoes** [comment]
@tumbili it broke because of my refactor. Don't sink too much work into debugging it.

File: src/modules/mc_pos_control/mc_pos_control_main.cpp [hunk 1]
```diff
@@ -1565,6 +1617,40 @@ void MulticopterPositionControl::control_auto(float dt)
...
+			_thrust_int(2) = -math::constrain(_thrust_int(2), _params.thr_min, _params.thr_max);
+			_landed_locked = false;
+		}
+	}
+
+	if (_takeoff_jumped) {
+		// reset previous and current velocity setpoint, now the position controller can take over
+		_vel_sp(2) = -_params.tko_speed;
+		_vel_sp_prev(2) = -_params.tko_speed;
+	}
```
Comment (Stifael) [inline src/modules/mc_pos_control/mc_pos_control_main.cpp]: still dont get why we set it to tko_speed and not to the current velocity (I am aware the difference might not be significant, but does it not make more sense to the set it to the actual value)

**Comment by Stifael** [comment]
I might be wrong, but I think this new version does not include the newest commits from #6035.

**Comment by julianoes** [comment]
I'm closing this in favor of #6205.

**Pull request closed without merging**